pipelines/jobs/configurations/jdk15u_pipeline_config.groovy

```diff
@@ -51,7 +51,7 @@ class Config15 {
                         hotspot: 'win2012&&vs2017'
                 ],
                 buildArgs : [
-                        hotspot : '--jvm-variant client,server'
```

Comment:
I believe that the default is server, Therefore you should be able to completely remove the buildArgs block.

Reply:
Removed.